src/amber/router/params.cr

```diff
@@ -61,13 +61,11 @@ module Amber::Router
     end
 
     def to_h
-      {
-        :multipart => multipart.to_h,
-        :json      => json.to_h,
-        :route     => route.to_h,
-        :form      => form.to_h,
-        :query     => query.to_h,
-      }
+      query.to_h
+        .merge(form.to_h)
```

Comment:
This should probably be memoed?

Reply:
I think is better not to memoize in case a value is changed or added